.php-cs-fixer.php

```diff
@@ -5,6 +5,8 @@
 $config = new PhpCsFixer\Config();
 $config->setCacheFile(__DIR__ . '/php-cs-fixer.cache');
 $config->setFinder($finder);
+$config->setIndent("\t");
```

Comment:
Hmmmm! Let's minimize refactor noise and honor existing style of 4 spaces here (Also, it's [PSR-12](https://www.php-fig.org/psr/psr-12/#24-indenting))

Reply:
Changed